src/reducers/locations.js

```diff
@@ -40,6 +42,20 @@ function constructNewStateWithUpdatedPhones(state, action, newPhones, location,
   };
 }
 
+export const locationErrorsReducer = (state = {}, action) => {
```

Comment:
Any reason to keep 2 high-level reducers in the same file?

Reply:
(I mean, I think keeping it in the same file could make sense, but then also probably keeping it in the same reducer, as just an "error" property and a "data" or whatnot property in the same object, to avoid the 2 switches listening to effectively the same actions and managing closely related state.)

Reply:
I don't want to do the refactoring that would need to update all the call sites that reference state.location, so that state.location maps `locationId -> {locationData, errorData}`, instead of `locationId -> locationData`. I am a bit concerned about breaking some existing behavior. I would therefore rather break errors out into a separate top-level state.

Reply:
Actually, I took another look, and refactoring does not seem too difficult. I'll go ahead and do this.

Reply:
I made a first pass at refactoring, and while all of the view code is easily updatable, the reducer code that handles this is non-trivial. Because we do not yet have automated tests, it is challenging to do a refactoring of the state without introducing bugs. So, I am going to go back to my original suggestion, which is that we do not change the locations data structure at this time, and instead introduce a new top-level state for errors.

Reply:
Yup, makes sense to keep it as separate high-level state then.
Back to the original question, though: any reason why the 2 reducers should be in the same file? (As opposed to a src/reducers/errors or src/reducers/locationErrors or something?)

Reply:
Cool. I think the convention of having a reducer per file is generally a good one.
(If only for consistency.)

Reply:
This is done in 160bb60c6149c94ec0dcc2dba5dcb96dec0ac629